proj/src/main/webapp/script.js

```diff
@@ -50,17 +50,96 @@ function errorCallback(err){
 function getLocation() {
     if(navigator.geolocation){
         navigator.geolocation.getCurrentPosition(successCallback,errorCallback,{timeout:10000, enableHighAccuracy:false});
-    }
-    else{
+    } else{
         console.log("Error. Geolocation not supported or not enabled");
     }
     return;
 }
 
+/*
+    Setting password 
+*/ 
+
+// Check if the password meets the password requirement with every character
+$('#pass').on('keyup', function(){
+    meetRequirements();
+    pwLength.hidden = false;
+    pwNumber.hidden = false;
+    pwSymbol.hidden = false;
+});
+
+// Check if the password is strong enough 
+function meetRequirements(){
+    const password = document.getElementById("pass");
+    var goodPassword = false;
```

Comment:
this `var goodPassword=false;` also can be removed.

Reply:
deleted